src/components/Dashboard/UserMenu.jsx

```diff
@@ -34,6 +34,12 @@ import SignInDialog from '../SignInDialog';
   leftIcon: {
     marginRight: theme.spacing.unit,
   },
+  usernameStyle : {
+    overflow: 'hidden',
+    'white-space': 'nowrap',
```

Comment:
Rename `white-space` to `whiteSpace` and remove the quotes.

```jsx
whiteSpace: 'nowrap',
```

Reply:
sorry for that. I am really new in version control

Reply:
No worries at all!